sale_invoice_line_note/models/sale_order.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models, api
+
+
+class SaleOrder(models.Model):
+
+    _inherit = 'sale.order'
+
+    @api.multi
+    def action_invoice_create(self, grouped=False, final=False):
+        """Create invoice note lines with notes from the sale order"""
```

Comment:
Question: note lines are not copied because of this line? https://github.com/odoo/odoo/blob/12.0/addons/sale/models/sale.py#L535 (0 qty_to_invoice)

Reply:
I'm not sure if it's only this line, but I guess we shouldn't set a `qty_to_invoice` anyway...